imagetest/testworkflow.go

```diff
@@ -346,6 +341,7 @@ func NewTestWorkflow(name, image, timeout string) (*TestWorkflow, error) {
 	t.wf = daisy.New()
 	t.wf.Name = strings.ReplaceAll(name, "_", "-")
 	t.wf.DefaultTimeout = timeout
+	t.osUnderTest = osUnderTest
```

Comment:
I also thought about just basing this variable off of the Image name. i.e.
```
	t.osUnderTest = "linux"
	if string.Contains(t.Image, "windows") {
		t.osUnderTest = "windows"
	}
```

This would allow us to avoid adding a new flag for imagetest and keep things a little tighter. But since it is doing string comparison against strings it'll be fairly brittle.

As it is having a new flag would not allow running imagetests against both linux and windows images at the same time.

Thoughts?

Reply:
i think it should deduce rather than require explicit input where possible. practically we are unlikely to need to test windows and linux at the same time however, so it isn't a hard blocker